lighthouse-cli/test/smokehouse/tricky-metrics/expectations.js

```diff
@@ -15,11 +15,11 @@ module.exports = [
       finalUrl: 'http://localhost:10200/tricky-tti.html',
       audits: {
         'first-cpu-idle': {
-          score: '<75',
+          // stalls for 5 seconds, 5 seconds out, so should be around 10s
```

Comment:
is there any harm in having `score` and `numericValue`? It's kind of nice to have a `first-cpu-idle` score curve assertion, even if it's only a single (very fuzzy) data point

Reply:
sure, fine with me. `<75` is definitely useless though 😆

Reply:
> sure, fine with me. <75 is definitely useless though 😆

lol, didn't notice the lack of a decimal point

...so do you want to do the score as well? I can't tell :)

We could tighten them up if we're this confident about the numericValues:
`'>9000'` for first-cpu-idle means a score of `'<.25'`
`'>9000'` for interactive means a score of `'<.35'`
`'>12000'` for interactive means a score of `'<.17'`

Reply:
> ...so do you want to do the score as well?

personally, I don't. I think it creates extra noise when the point of the tests was to assert on a particular metric value and not specifics of our scoring curve. I concede it makes sense to have at least a few score assertions to make sure we're not going crazy there, but IMO the unit tests are good enough there and we don't need to copy it everywhere.

I don't feel strongly enough to fight it on what was supposed to be a quick feedback followup PR though so 😆  🤷‍♂

Reply:
haha, fair enough